apps/web/src/features/cohorts/components/TeacherCohorts.tsx

```
@@ -0,0 +1,573 @@
+							<TableHead>Weekly Sessions</TableHead>
+							<TableHead>Teachers</TableHead>
+							<TableHead>Status</TableHead>
+							<TableHead>Start Date</TableHead>
+						</TableRow>
+					</TableHeader>
+					<TableBody>
+						{loadingStats ? (
+							// Loading stats skeletons
+							Array.from({ length: 3 }).map((_, i) => (
+								<TableRow key={i}>
+									<TableCell><Skeleton className="h-4 w-40" /></TableCell>
```

Biome (2.1.2)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)